CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 Changelog
 =========
 
-v6.0.0-alpha.15 - 2016-01-18
+v6.0.0-alpha.16 - 2017-02-15
```

Comment:
I don't consider this critical for this release (don't want to block on something like this), but in the future, can we trail each line item with an Issue # where applicable? I'd like for people to be able to trackback to context for each line item.